modules/lang-painless/src/main/java/org/elasticsearch/painless/ScriptInterface.java

```diff
@@ -34,19 +35,24 @@
  * Information about the interface being implemented by the painless script.
  */
 public class ScriptInterface {
```

Comment:
This entire class name doesn't really make sense anymore. Maybe something like `ScriptClassInfo`?